src/spec/payid-protocol.md

```diff
@@ -503,7 +498,7 @@ This specification defines the PayID protocol - an application-layer protocol, w
   PayID protocol MUST NOT be used to provide payment account(s) information corresponding to a PayID URI unless providing that data via PayID protocol by the relevant PayID server was explicitly authorized by the PayID owner. If a PayID owner wishes to limit access to information, PayID servers MAY provide an interface by which PayID owners can select which information is exposed through the PayID server interface. For example, PayID servers MAY allow PayID owners to mark certain data as `public` and then utilize that marking as a means of determining what information to expose via PayID protocol. The PayID servers MAY also allow PayID owners to provide a whitelist of users who are authorized to access the specific information. In such a case, the PayID server MUST authenticate the PayID client.
 
 ## Payment Address Rotation
-  The power of PayID protocol comes from providing a single place where others can find payment account(s) information corresponding to a PayID URI, but PayID owners should be aware of how easily payment account information that one might publish can be used in unintended ways. As one example, one might query a PayID server only to see if a given PayID URI is valid and if so, get the list of associated payment account information. If the PayID server uses the same payment address each time, it becomes easy for a third-party to track one's entire payment history. The PayID server MUST follow the best practice of payment address rotation for every query to mitigate this privacy concern.
+  The power of PayID protocol comes from providing a single place where others can find payment account(s) information corresponding to a PayID URI, but PayID owners should be aware of how easily payment account information that one might publish can be used in unintended ways. As one example, one might query a PayID server only to see if a given PayID URI is valid and if so, get the list of associated payment account information. If the PayID server uses the same payment address each time, it becomes easy for a third-party to track one's entire payment history. The PayID server thus MAY follow the best practice of payment address rotation on every query for cryptocurrencies addresses to mitigate this privacy concern.
```

Comment:
Originally: `The PayID server MUST follow the best practice of payment address rotation for every query to mitigate this privacy concern.`

Now: `The PayID server thus MAY follow the best practice of payment address rotation on every query for cryptocurrencies addresses to mitigate this privacy concern.`

It seems reasonable for an RFC to talk about best practices, but it's absolutely not a "MUST" that PayID servers rotate crypto address tags. You can implement the PayID Protocol without following best practices.

Reply:
I agree. +1 to the `MAY` adjustment.

Also, a common best-practice (especially in Bitcoin) is to have a well-known receive address that never changes, with BTC being periodically "swept" to a more secure bitcoin account (sometimes at an exchange).

Also, in the case of XRPL, once XRP is sent to an exchange, there is generally no correlation between the accounts that receive XRP and the accounts that "send out" XRP, so in many ways the exchanges are providing the actual desired outcome of this "MAY" without PayID servers really having to do anything.